### DISCUSSION TITLE
Album Banners

### DISCUSSION THREAD

**Discussion by SoNothingMC** [ciderapp/Cider#740] [Added Features]
I honestly have no idea if this can be executed or not lol

But I'm basically talking about these:  

![image](https://user-images.githubusercontent.com/32032824/160653579-1eb34274-21d4-43a2-85f5-e630f63f5962.png)  
![image](https://user-images.githubusercontent.com/32032824/160653608-b4b4bacc-13c7-451e-bf08-ca8002e631e7.png)

Is there any possibility that they can be added into Cider (and would it look good anyways with the Cider UI)

**Comment by KillerParticles** [comment]
Something like this concept I did?
![image](https://user-images.githubusercontent.com/93286201/160661267-c836c026-eeee-4c36-a812-a27d5a29e292.png)

**Comment by SoNothingMC** [comment]
Well would you look at that!  
I implemented my suggestion myself! (well, ~50% of it, big thanks to vapormusic for the help!)

See #1196 for more info!

![A screenshot of the Cider app, showing the EP "101 / Yakou" by Sangatsu no Phantasia, along with a banner for the album.](https://user-images.githubusercontent.com/32032824/175781799-be88a85e-fe06-4318-800d-b2e569b29043.png) <p align="center"><i>Final result</i></p>

**Comment by KillerParticles** [comment]
Looks one to one with my concept lmao, well done! It looks amazing.

**Comment by Amaru8** [comment]
Added in #1196, locking discussion.
